src/ITensorsExt/itensorutils.jl

```diff
@@ -19,3 +19,32 @@ inv_diag(it::ITensor) = map_diag(inv, it)
 invsqrt_diag(it::ITensor) = map_diag(inv ∘ sqrt, it)
 pinv_diag(it::ITensor) = map_diag(pinv, it)
 pinvsqrt_diag(it::ITensor) = map_diag(pinv ∘ sqrt, it)
+
+"""Given a vector of ITensors, separate them into groups of commuting itensors (i.e. itensors in the same group do not share any common indices)"""
+function group_ITensors(its::Vector{ITensor})
```

Comment:
Change to all lower case, i.e. `group_itensors`. The convention in Julia is that function names are all lower case. Only types, constructors, and module names have capital letters.

Let's think of a more specific name for this, for example something that specifies _how_ the ITensors are being grouped, since they could be grouped in many different ways.